src/main/java/io/logz/jmx2graphite/Jmx2GraphiteConfiguration.java

```diff
@@ -133,17 +136,19 @@ private GraphiteProtocol getGraphiteProtocol(Config config) {
 
     private void setFilterPatterns(Config config) {
         try {
-            whiteListPattern = config.hasPath(Jmx2GraphiteJavaAgent.WHITE_LIST_REGEX_CONF_JOLOKIA) ?
-                    Pattern.compile(config.getString(Jmx2GraphiteJavaAgent.WHITE_LIST_REGEX_CONF_JOLOKIA)) : null;
-        } catch (Exception e) {
-            whiteListPattern = null;
+            whiteListPattern = Optional.of(Pattern.compile(config.getString("filter.whitelistRegEx")));
```

Comment:
The name of the method is wrong. perhaps: `parseFilterPatterns(Config config)` ?